No javavi library classes found, it means that we couldn't compile it. Do you have JDK7+ installed?

This appear in my Windows 7 with JDK8. I didn't install my JDK in the default path, but another place.
This is my jdk configure:

JAVA_HOME = C:\bin\Java\jdk1.8.0_66
CLASSPATH = .;%JAVA_HOME%\lib\dt.jar;%JAVA_HOME%\lib\tools.jar;
PATH = C:\ProgramData\Oracle\Java\javapath;%JAVA_HOME%\bin;%JAVA_HOME%\jre\bin;C:\bin\Vim74\vim74\;

Please help me a little.

C:\Windows\system32\cmd.exe /c (javac -d C:/Users/zhaochenyou/.vim/bundle/vim-javacomplete
2/libs/javavi/target/classes -classpath C:/Users/zhaochenyou/.vim/bundle/vim-javacomplete2
/libs/javavi/target/classes:C:/Users/zhaochenyou/.vim/bundle/vim-javacomplete2/libs/javapa
rser.jar; -sourcepath C:/Users/zhaochenyou/.vim/bundle/vim-javacomplete2/libs/javavi/src/m
ain/java: -g -nowarn -target 1.8 -source 1.8 -encoding UTF-8 C:/Users/zhaochenyou/.vim/bun
dle/vim-javacomplete2/libs/javavi/src/main/java/kg/ash/javavi/Javavi.java)
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:5: error: package kg.ash.javavi.actions does not exist
import kg.ash.javavi.actions.Action;
                            ^
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:6: error: package kg.ash.javavi.actions does not exist
import kg.ash.javavi.actions.ActionFactory;
                            ^
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:7: error: package kg.ash.javavi.clazz does not exist
import kg.ash.javavi.clazz.SourceClass;
                          ^
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:8: error: package kg.ash.javavi.searchers does not exist
import kg.ash.javavi.searchers.ClassMap;
                              ^
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:16: error: cannot find symbol
    public static HashMap<String,SourceClass> cachedClasses = new HashMap<>();
                                 ^
  symbol:   class SourceClass
  location: class Javavi
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:18: error: cannot find symbol
    public static HashMap<String,ClassMap> cachedClassPackages = new HashMap<>();
                                 ^
  symbol:   class ClassMap
  location: class Javavi
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:49: error: cannot find symbol
    public static Daemon daemon = null;
                  ^
  symbol:   class Daemon
  location: class Javavi
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:60: error: cannot find symbol
        Action action = null;
        ^
  symbol:   class Action
  location: class Javavi
C:\Users\zhaochenyou\.vim\bundle\vim-javacomplete2\libs\javavi\src\main\java\kg\ash\javavi
\Javavi.java:82: error: cannot find symbol
                        action = ActionFactory.get(arg);
                                 ^
  symbol:   variable ActionFactory
  location: class Javavi
9 errors
shell returned 1
Hit any key to close this window...

--- a/autoload/javacomplete/server.vim
+++ b/autoload/javacomplete/server.vim
@@ -126,7 +126,7 @@ function! javacomplete#server#Compile()
     exe '!'. 'mvn -f "'. javaviDir . g:FILE_SEP . 'pom.xml" compile'
   else
     call mkdir(javaviDir. join(['target', 'classes'], g:FILE_SEP), "p")
-    exe '!'. javacomplete#server#GetCompiler(). ' -d '. javaviDir. 'target'. g:FILE_SEP. 'classes -classpath '. javaviDir. 'target'. g:FILE_SEP. 'classes'. g:PATH_SEP. g:JavaComplete_Home. g:FILE_SEP .'libs'. g:FILE_SEP. 'javaparser.jar'. g:PATH_SEP. ' -sourcepath '. javaviDir. 'src'. g:FILE_SEP. 'main'. g:FILE_SEP. 'java -g -nowarn -target 1.8 -source 1.8 -encoding UTF-8 '. javaviDir. join(['src', 'main', 'java', 'kg', 'ash', 'javavi', 'Javavi.java'], g:FILE_SEP)
+    exe '!'. javacomplete#server#GetCompiler(). ' -d "'. javaviDir. 'target'. g:FILE_SEP. 'classes" -classpath "'. javaviDir. 'target'. g:FILE_SEP. 'classes'. g:PATH_SEP. g:JavaComplete_Home. g:FILE_SEP .'libs'. g:FILE_SEP. 'javaparser.jar" -sourcepath "'. javaviDir. 'src'. g:FILE_SEP. 'main'. g:FILE_SEP. 'java" -g -nowarn -target 1.8 -source 1.8 -encoding UTF-8 "'. javaviDir. join(['src', 'main', 'java', 'kg', 'ash', 'javavi', 'Javavi.java"'], g:FILE_SEP)
   endif
 endfunction
 

